openapi/generator.go

```diff
@@ -104,6 +104,12 @@ func (g *Generator) SetServers(servers []*Server) {
 	g.api.Servers = servers
 }
 
+// SetSecurity sets the security options for the
+// current specification.
+func (g *Generator) SetSecurity(security *SecurityRequirement) {
```

Comment:
I would prefer to name the function `SetSecurityRequirement`, to be in-par with the OpenAPI spec object naming.